system/cmd_runner_interface.go

```diff
@@ -18,6 +18,9 @@ type Command struct {
 	// Don't echo stdout/stderr
 	Quiet bool
 
+	// Run command with a priority lower than the parent process
+	RunNicer bool
```

Comment:
Is there perhaps a better name for this field indicating what nicer means? From skimming the linked issue (and the comment!) but it would not be obvious to me from the method name here. 

Maybe something like: “ReducePriority” or “SpawnWithLowerPriority” or something like that? I don’t have a great suggestion. I’m sure there’s a standard name for this thing but I can’t remember off the top of my head what it should be.